packages/chips/ChipSet.js

```diff
@@ -65,54 +73,39 @@ export default class ChipSet extends Component {
     return {
       hasClass: (className) => this.classes.split(' ').includes(className),
       setSelected: (chipId, selected) => {
-        const {selectedChipIds} = this.state;
-        if (selected) {
-          selectedChipIds.add(chipId);
-        } else {
-          selectedChipIds.delete(chipId);
-        }
-        this.setState({selectedChipIds});
+        const selectedChipIds = this.state.foundation.getSelectedChipIds();
+        this.setState({selectedChipIds}, () => {
```

Comment:
remove `this.setState` since we're not keeping `selectedChipIds` in state anymore